install/web-check-install.sh

```diff
@@ -0,0 +1,159 @@
+  g++ \
+  traceroute \
+  xvfb \
+  dbus \
+  xorg \
+  xvfb \
+  gtk2-engines-pixbuf \
+  dbus-x11 \
+  xfonts-base \
+  xfonts-100dpi \
+  xfonts-75dpi \
+  xfonts-scalable \
+  imagemagick \
+  x11-apps
+msg_ok "Installed Dependencies"
+
+msg_info "Setting up Node.js Repository"
+mkdir -p /etc/apt/keyrings
+curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg
+echo "deb [signed-by=/etc/apt/keyrings/nodesource.gpg] https://deb.nodesource.com/node_22.x nodistro main" >/etc/apt/sources.list.d/nodesource.list
+msg_ok "Set up Node.js Repository"
+
+msg_info "Setup Python3"
+$STD apt-get install -y python3
+rm -rf /usr/lib/python3.*/EXTERNALLY-MANAGED
+msg_ok "Setup Python3"
+
+msg_info "Installing Chromium"
+curl -fsSL https://dl-ssl.google.com/linux/linux_signing_key.pub | gpg --dearmor -o /etc/apt/trusted.gpg.d/google-archive.gpg
+echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >/etc/apt/sources.list.d/google.list
+$STD apt-get update
+$STD apt-get -y install \
+  chromium \
+  libxss1 \
+  lsb-release
+msg_ok "Installed Chromium"
+
+msg_info "Installing Node.js"
+$STD apt-get update
+$STD apt-get install -y nodejs
+$STD npm install -g yarn
+msg_ok "Installed Node.js"
+
+msg_info "Setting up Chromium"
+/usr/bin/chromium --no-sandbox --version > /etc/chromium-version
+chmod 755 /usr/bin/chromium
+msg_ok "Finished setting up Chromium"
+
+msg_info "Installing Web-Check (Patience)"
+temp_file=$(mktemp)
+RELEASE="patch-1"
+wget -q "https://github.com/CrazyWolf13/web-check/archive/refs/heads/${RELEASE}.tar.gz" -O $temp_file
+tar xzf $temp_file
+mv web-check-${RELEASE} /opt/web-check
+cd /opt/web-check
+cat > /opt/web-check/.env << 'EOF'
+CHROME_PATH=/usr/bin/chromium
+PUPPETEER_EXECUTABLE_PATH=/usr/bin/chromium
+HEADLESS=true
+GOOGLE_CLOUD_API_KEY=''
+REACT_APP_SHODAN_API_KEY=''
+REACT_APP_WHO_API_KEY=''
+SECURITY_TRAILS_API_KEY=''
+CLOUDMERSIVE_API_KEY=''
+TRANCO_USERNAME=''
+TRANCO_API_KEY=''
+URL_SCAN_API_KEY=''
+BUILT_WITH_API_KEY=''
+TORRENT_IP_API_KEY=''
+PORT='3000'
+DISABLE_GUI='false'
+API_TIMEOUT_LIMIT='10000'
+API_CORS_ORIGIN='*'
+API_ENABLE_RATE_LIMIT='false'
+REACT_APP_API_ENDPOINT='/api'
+ENABLE_ANALYTICS='false'
+EOF
+$STD yarn install --frozen-lockfile --network-timeout 100000
+echo "${RELEASE}" >/opt/${APPLICATION}_version.txt
+msg_ok "Installed Web-Check"
+
+msg_info "Building Web-Check"
+$STD yarn build --production
+msg_ok "Built Web-Check"
+
+msg_info "Creating Service"
+cat > /opt/run_web-check.sh << 'EOF'
+#!/bin/bash
+SCREEN_RESOLUTION="1280x1024x24"
+if ! systemctl is-active --quiet dbus; then
+  echo "Warning: dbus service is not running. Some features may not work properly."
+fi
+[[ -z "${DISPLAY}" ]] && export DISPLAY=":99"
+Xvfb "${DISPLAY}" -screen 0 "${SCREEN_RESOLUTION}" &
+XVFB_PID=$!
+sleep 2
+cd /opt/web-check
+exec yarn start
+EOF
+chmod +x /opt/run_web-check.sh
+cat > /etc/systemd/system/web-check.service << 'EOF'
```

Comment:
```suggestion
cat <<EOF >/etc/systemd/system/web-check.service
```
Nitpicking here, but all scripts do it like this
For the sake of the contributor docs revision (pending), we should do it like this

Reply:
@tremor021 Thanks, apologies, me being lazy and not looking properly at the others :D